lib/taskana-core/src/main/java/pro/taskana/exceptions/ClassificationNotFoundException.java

```diff
@@ -1,6 +1,7 @@
 package pro.taskana.exceptions;
 
 /**
+<<<<<<< HEAD
  * This exception will be thrown if a specific task is not in the database.
  */
```

Comment:
Javadoc should say: ..thrown if a specific Classification is not in the database